project.clj

```diff
@@ -30,6 +34,8 @@
   :classifiers [["test" :testutils]]
 
   :profiles {:dev {:dependencies [[spyscope "0.1.4"]
```

Comment:
FYI we don't need to put `spyscope` in every single project anymore, you can leave this in your personal `profiles.clj`.

Doesn't hurt being here, but we're just dragging this stuff around and I've not heard of anyone using this other than Chris.